schematron/v2.2.0/tests/test_v2-2-0_L200_Audit.py

```diff
@@ -159,3 +159,140 @@ def test_all_fenestration_tests_are_run(self):
 
         # -- Assert
         self.assert_failure_messages(failures, {})
+
+    def test_is_invalid_when_ground_coupling_is_slab_on_grade_and_invalid(self):
+        # -- Setup
+        tree = exemplary_tree('L200_Audit', 'v2.2.0')
+
+        # replace the ground coupling with another valid ground coupling
+        new_coupling_xml = f"""
```

Comment:
does this need to be added since it's already in the L200_Audit.xml exemplary file?

Reply:
I just added it to be complete, in case the exemplary file was changed at some point. It's not totally necessary but I don't see the harm in keeping it